pkg/render/manager.go

```diff
@@ -600,6 +606,63 @@ func (c *managerComponent) voltronContainer() corev1.Container {
 	}
 }
 
+// dashboardContainer returns the dashboard sidecar container that only gets created in Enterprise (where tenancy is
+// not enabled).
+func (c *managerComponent) dashboardContainer() corev1.Container {
+	var keyPath, certPath string
+	if c.cfg.InternalTLSKeyPair != nil {
+		keyPath, certPath = c.cfg.InternalTLSKeyPair.VolumeMountKeyFilePath(), c.cfg.InternalTLSKeyPair.VolumeMountCertificateFilePath()
+	}
+
+	env := []corev1.EnvVar{
+		{Name: "LISTEN_ADDR", Value: fmt.Sprintf("127.0.0.1:%s", DashboardAPIPort)},
+		{Name: "LOG_LEVEL", Value: "Info"},
+		{Name: "LINSEED_URL", Value: fmt.Sprintf("https://tigera-linseed.%s.svc.%s", ElasticsearchNamespace, c.cfg.ClusterDomain)},
+		{Name: "LINSEED_CLIENT_KEY", Value: keyPath},
+		{Name: "LINSEED_CLIENT_CERT", Value: certPath},
+		{Name: "MULTI_CLUSTER_FORWARDING_ENDPOINT", Value: fmt.Sprintf("https://tigera-manager.%s.svc:%s", c.cfg.Namespace, defaultVoltronPort)},
+		{Name: "HEALTH_PORT", Value: DashboardAPIHealthPort},
+	}
+
+	mounts := append(
+		c.cfg.TrustedCertBundle.VolumeMounts(c.SupportedOSType()),
+		c.cfg.InternalTLSKeyPair.VolumeMount(c.SupportedOSType()),
+	)
+
+	return corev1.Container{
+		Name:            DashboardAPIName,
+		Image:           c.uiAPIsImage,
+		ImagePullPolicy: ImagePullPolicy(),
+		Command:         []string{"/usr/bin/dashboard-api"},
+		Env:             env,
+		VolumeMounts:    mounts,
+		SecurityContext: securitycontext.NewNonRootContext(),
+		ReadinessProbe: &corev1.Probe{
+			ProbeHandler: corev1.ProbeHandler{
+				Exec: &corev1.ExecAction{
+					Command: []string{"/usr/bin/dashboard-api", "-ready"},
+				},
+			},
+			FailureThreshold:    3,
```

Comment:
We have code in components.go that sents defaults for our probes `setProbeTimeouts`. I was just wondering if these settings you hard-coded are copied over from kubernetes defaults or if they need to be set specifically, like they have here.

Reply:
They are copied over from `cc-mgmt-core` that installs custom dashboards in CC.